# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1625567078 -7200
# Parent  a3b5e92bc304ac0069c0664722d90a6faee9809d
Branding 1: Mostly `hg copy mail/branding/thunderbird/ mail/branding/betterbird` with the exception of some Mac files, see below:

We don't copy LINCESE and the Mac files dsstore, disk.icns, thunderbird.icns, background.png

diff --git a/mail/branding/thunderbird/TB-symbolic.svg b/mail/branding/betterbird/TB-symbolic.svg
copy from mail/branding/thunderbird/TB-symbolic.svg
copy to mail/branding/betterbird/TB-symbolic.svg
diff --git a/mail/branding/thunderbird/VisualElements_150.png b/mail/branding/betterbird/VisualElements_150.png
copy from mail/branding/thunderbird/VisualElements_150.png
copy to mail/branding/betterbird/VisualElements_150.png
diff --git a/mail/branding/thunderbird/VisualElements_70.png b/mail/branding/betterbird/VisualElements_70.png
copy from mail/branding/thunderbird/VisualElements_70.png
copy to mail/branding/betterbird/VisualElements_70.png
diff --git a/mail/branding/thunderbird/addressbook.ico b/mail/branding/betterbird/addressbook.ico
copy from mail/branding/thunderbird/addressbook.ico
copy to mail/branding/betterbird/addressbook.ico
diff --git a/mail/branding/thunderbird/branding.nsi b/mail/branding/betterbird/branding.nsi
copy from mail/branding/thunderbird/branding.nsi
copy to mail/branding/betterbird/branding.nsi
diff --git a/mail/branding/thunderbird/configure.sh b/mail/branding/betterbird/configure.sh
copy from mail/branding/thunderbird/configure.sh
copy to mail/branding/betterbird/configure.sh
diff --git a/mail/branding/thunderbird/content/about-logo.svg b/mail/branding/betterbird/content/about-logo.svg
copy from mail/branding/thunderbird/content/about-logo.svg
copy to mail/branding/betterbird/content/about-logo.svg
diff --git a/mail/branding/thunderbird/content/about-wordmark.svg b/mail/branding/betterbird/content/about-wordmark.svg
copy from mail/branding/thunderbird/content/about-wordmark.svg
copy to mail/branding/betterbird/content/about-wordmark.svg
diff --git a/mail/branding/thunderbird/content/about.png b/mail/branding/betterbird/content/about.png
copy from mail/branding/thunderbird/content/about.png
copy to mail/branding/betterbird/content/about.png
diff --git a/mail/branding/thunderbird/content/aboutDialog.css b/mail/branding/betterbird/content/aboutDialog.css
copy from mail/branding/thunderbird/content/aboutDialog.css
copy to mail/branding/betterbird/content/aboutDialog.css
diff --git a/mail/branding/thunderbird/default128.png b/mail/branding/betterbird/default128.png
copy from mail/branding/thunderbird/default128.png
copy to mail/branding/betterbird/default128.png
diff --git a/mail/branding/thunderbird/default16.png b/mail/branding/betterbird/default16.png
copy from mail/branding/thunderbird/default16.png
copy to mail/branding/betterbird/default16.png
diff --git a/mail/branding/thunderbird/default22.png b/mail/branding/betterbird/default22.png
copy from mail/branding/thunderbird/default22.png
copy to mail/branding/betterbird/default22.png
diff --git a/mail/branding/thunderbird/default24.png b/mail/branding/betterbird/default24.png
copy from mail/branding/thunderbird/default24.png
copy to mail/branding/betterbird/default24.png
diff --git a/mail/branding/thunderbird/default256.png b/mail/branding/betterbird/default256.png
copy from mail/branding/thunderbird/default256.png
copy to mail/branding/betterbird/default256.png
diff --git a/mail/branding/thunderbird/default32.png b/mail/branding/betterbird/default32.png
copy from mail/branding/thunderbird/default32.png
copy to mail/branding/betterbird/default32.png
diff --git a/mail/branding/thunderbird/default48.png b/mail/branding/betterbird/default48.png
copy from mail/branding/thunderbird/default48.png
copy to mail/branding/betterbird/default48.png
diff --git a/mail/branding/thunderbird/default64.png b/mail/branding/betterbird/default64.png
copy from mail/branding/thunderbird/default64.png
copy to mail/branding/betterbird/default64.png
diff --git a/mail/branding/thunderbird/jar.mn b/mail/branding/betterbird/jar.mn
copy from mail/branding/thunderbird/jar.mn
copy to mail/branding/betterbird/jar.mn
diff --git a/mail/branding/thunderbird/locales/Makefile.in b/mail/branding/betterbird/locales/Makefile.in
copy from mail/branding/thunderbird/locales/Makefile.in
copy to mail/branding/betterbird/locales/Makefile.in
diff --git a/mail/branding/thunderbird/locales/en-US/brand.dtd b/mail/branding/betterbird/locales/en-US/brand.dtd
copy from mail/branding/thunderbird/locales/en-US/brand.dtd
copy to mail/branding/betterbird/locales/en-US/brand.dtd
diff --git a/mail/branding/thunderbird/locales/en-US/brand.ftl b/mail/branding/betterbird/locales/en-US/brand.ftl
copy from mail/branding/thunderbird/locales/en-US/brand.ftl
copy to mail/branding/betterbird/locales/en-US/brand.ftl
diff --git a/mail/branding/thunderbird/locales/en-US/brand.properties b/mail/branding/betterbird/locales/en-US/brand.properties
copy from mail/branding/thunderbird/locales/en-US/brand.properties
copy to mail/branding/betterbird/locales/en-US/brand.properties
diff --git a/mail/branding/thunderbird/locales/jar.mn b/mail/branding/betterbird/locales/jar.mn
copy from mail/branding/thunderbird/locales/jar.mn
copy to mail/branding/betterbird/locales/jar.mn
diff --git a/mail/branding/thunderbird/locales/moz.build b/mail/branding/betterbird/locales/moz.build
copy from mail/branding/thunderbird/locales/moz.build
copy to mail/branding/betterbird/locales/moz.build
diff --git a/mail/branding/thunderbird/messengerWindow.ico b/mail/branding/betterbird/messengerWindow.ico
copy from mail/branding/thunderbird/messengerWindow.ico
copy to mail/branding/betterbird/messengerWindow.ico
diff --git a/mail/branding/thunderbird/moz.build b/mail/branding/betterbird/moz.build
copy from mail/branding/thunderbird/moz.build
copy to mail/branding/betterbird/moz.build
diff --git a/mail/branding/thunderbird/net.thunderbird.Thunderbird.appdata.xml b/mail/branding/betterbird/net.thunderbird.Thunderbird.appdata.xml
copy from mail/branding/thunderbird/net.thunderbird.Thunderbird.appdata.xml
copy to mail/branding/betterbird/net.thunderbird.Thunderbird.appdata.xml
diff --git a/mail/branding/thunderbird/newmail.ico b/mail/branding/betterbird/newmail.ico
copy from mail/branding/thunderbird/newmail.ico
copy to mail/branding/betterbird/newmail.ico
diff --git a/mail/branding/thunderbird/thunderbird-branding.js b/mail/branding/betterbird/thunderbird-branding.js
copy from mail/branding/thunderbird/thunderbird-branding.js
copy to mail/branding/betterbird/thunderbird-branding.js
diff --git a/mail/branding/thunderbird/thunderbird.VisualElementsManifest.xml b/mail/branding/betterbird/thunderbird.VisualElementsManifest.xml
copy from mail/branding/thunderbird/thunderbird.VisualElementsManifest.xml
copy to mail/branding/betterbird/thunderbird.VisualElementsManifest.xml
diff --git a/mail/branding/thunderbird/wizHeader.bmp b/mail/branding/betterbird/wizHeader.bmp
copy from mail/branding/thunderbird/wizHeader.bmp
copy to mail/branding/betterbird/wizHeader.bmp
diff --git a/mail/branding/thunderbird/wizHeaderRTL.bmp b/mail/branding/betterbird/wizHeaderRTL.bmp
copy from mail/branding/thunderbird/wizHeaderRTL.bmp
copy to mail/branding/betterbird/wizHeaderRTL.bmp
diff --git a/mail/branding/thunderbird/wizWatermark.bmp b/mail/branding/betterbird/wizWatermark.bmp
copy from mail/branding/thunderbird/wizWatermark.bmp
copy to mail/branding/betterbird/wizWatermark.bmp
diff --git a/mail/branding/thunderbird/writeMessage.ico b/mail/branding/betterbird/writeMessage.ico
copy from mail/branding/thunderbird/writeMessage.ico
copy to mail/branding/betterbird/writeMessage.ico
